modules/lang-painless/src/main/java/org/elasticsearch/painless/WhitelistLoader.java

```diff
@@ -43,18 +43,29 @@
      * and field.  Most validation will be done at a later point after all white-lists have been gathered and their
      * merging takes place.
      *
+     * A painless type name is considered to one of the following:
```

Comment:
`is considered to one of` -> `is one of`

Reply:
Also, maybe make it clear this is explained so that these types can be used in method signatures. The class portion of the whitelist syntax must contain the fully qualified name.

Reply:
Done.